snappy-tools/build.gradle

```diff
@@ -103,7 +103,9 @@ task docs(type: ScalaDoc) {
         !it.getPath().matches('.*/internal/.*') && !it.getPath().contains('com/gemstone/gemfire/cache/operations/')
       })
       if (project.plugins.hasPlugin('scala')) {
-        allSource.addAll(project.sourceSets.main.allScala)
+        allSource.addAll(project.sourceSets.main.allScala.findAll {
+          !it.getPath().matches('.*org/apache/spark/sql/execution/joins/HashedRelation.*') 
+        })
       }
```

Comment:
What's the problem with scaladoc generation for HashedRelation and shouldn't that be fixed instead of excluding it? Spark should already be creating scaladoc for that in their maven build, so are their our changes causing an issue?